presentation/src/main/java/com/globant/equattrocchio/cleanarchitecture/util/bus/observers/getImageDetailButtonObserver.java

```diff
@@ -0,0 +1,15 @@
+package com.globant.equattrocchio.cleanarchitecture.util.bus.observers;
+
+public abstract class getImageDetailButtonObserver extends BusObserver<getImageDetailButtonObserver.GetImageDetailButtonPressed> {
```

Comment:
the name of the class should be with upperCase `ImageDetailButtonObserver` and without the get